zk/stages/stage_sequencer_executor_verify.go

```diff
@@ -80,8 +88,12 @@ func SpawnSequencerExecutorVerifyStage(
 	// we could be running in a state with no executors so we need instant response that we are in an
 	// ok state to save lag in the data stream !!Dragons: there will be no witnesses stored running in
 	// this mode of operation
-	canVerify := cfg.verifier.HasExecutors()
+	canVerify := cfg.verifier.HasExecutorsUnsafe()
 	if !canVerify {
+		hermezDbReader := hermez_db.NewHermezDbReader(tx)
+		if err = cfg.verifier.WriteBatchToStream(latestBatch, hermezDbReader, tx); err != nil {
+			return err
+		}
```

Comment:
just to double check, this is no longer done in the sequencer execute stage?

Reply:
It was done as soon as the executor confirms that the batch is valid. This case just covers the case when we have no external executor. I have never check the executor stage itself but I think it wasn't done there because this function was private ;)

Reply:
![image](https://github.com/0xPolygonHermez/cdk-erigon/assets/70711990/360b81f0-a146-46e7-9672-35af1bb0a3d0)
it's still being done in the executor stage, so we need to pick which approach we like best

Reply:
I would prefer doing it always in one place. What do you think - leave it only at verification stage?